examples/ahiqar-arabic-karshuni-local.html

```diff
@@ -72,6 +72,26 @@
                         all: true,
                       },
                       citationKey: "Citation",
+                      orderDocuments: ["manifest", "collection", "item"],
+                      orderCollectionMetadata: [
```

Comment:
As I stated in the issue comment, I think the "..Metadata" suffix is not needed.